### PR TITLE
Fixes for e2e tests

### PR DESCRIPTION
**Pull request by mjura**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**
Issue #

**Special notes for your reviewer**:

**Checklist**:

- [ ] squashed commits into logical changes
- [ ] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests
- [ ] backport needed

### CODE REVIEW

File: test/e2e/config/config.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ bridgeIP: 172.17.0.1
 operatorReplicas: 1
 artifactsDir: ../../_artifacts
 
-certManagerVersion: v1.9.2
+certManagerVersion: v1.11.1
 certManagerChartURL: https://charts.jetstack.io/charts/cert-manager-${CERT_MANAGER_VERSION}.tgz
 
-rancherVersion: 2.7.9
+rancherVersion: 2.8.2
```
Comment (salasberryfin) [inline test/e2e/config/config.yaml:11]: Are we using this version? I noticed there's a different one hardcoded when installing rancher.

Comment (mjura) [reply test/e2e/config/config.yaml:11]: this is Rancher helm chart version, with this Helm chart we are deploying latest image `v2.9-head`

File: test/e2e/suite_test.go [hunk 2]
```diff
@@ -154,6 +154,8 @@ var _ = BeforeSuite(func() {
 					"global.cattle.psp.enabled=false",
 					"--set", fmt.Sprintf("hostname=%s.%s", e2eCfg.ExternalIP, e2eCfg.MagicDNS),
 					"--create-namespace",
+					"--devel",
+					"--set rancherImage=v2.9-head",
```
Comment (furkatgofurov7) [inline test/e2e/suite_test.go:158]: same comment as in EKS operator: https://github.com/rancher/eks-operator/pull/376#discussion_r1502337953

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: test/e2e/suite_test.go (hunk 2 above)
Comment (mjura) [reply test/e2e/suite_test.go]: yep, there was a typo, fixed

**Review by salasberryfin — APPROVED** [review]

**Review by alexander-demicev — APPROVED** [review]

**Review by furkatgofurov7 — APPROVED** [review]

**Review by salasberryfin — APPROVED** [review]

**Pull request merged**